packages/buefy-next/src/components/rate/Rate.spec.ts

```diff
@@ -52,7 +54,7 @@ describe('BRate', () => {
     it('manage methods correctly', async () => {
         const event = {
             stopPropagation: jest.fn()
```

Comment:
Please replace `jest.fn` with [`vi.fn`](https://vitest.dev/api/vi.html#vi-fn). You may import `vi` from the `vitest` package.